engines/griffon/resources.cpp

```diff
@@ -293,26 +293,36 @@ void GriffonEngine::loadMap(int mapnum) {
 					rcDest.setWidth(16);
 					rcDest.setHeight(16);
 
+					int flag=0;
 					if (l == 2 && curtilel == 1) {
 						for (int ff = 0; ff <= 5; ff++) {
 							int ffa = 20 * 5 - 1 + ff * 20;
 							int ffb = 20 * 5 + 4 + ff * 20;
 							if (curtile > ffa && curtile < ffb) {
-								_tiles[curtilel]->setAlpha(128, true);
+								flag=1;
```

Comment:
Why don't you just instead of this flag do something like `alphaValue = 128` and then use it directly?

Reply:
According to the context, I think that his alpha rendering is triggered under certain conditions, and I hope that the logic of the source code is not changed when possible.